src/Tribe/Views/V2/Utils/Separators.php

```diff
@@ -77,4 +77,40 @@ public static function should_have_time( $events, $event ) {
 
 		return isset( $start_hours_ids[ $event_id ] );
 	}
+
+	/**
+	 * Determines if a given event from a list of events should have a type separator
+	 * for the day view template structure.
+	 *
+	 * @since  TBD
+	 *
+	 * @param  array       $events WP_Post or numeric ID for events.
+	 * @param  WP_Post|int $event  Event we want to determine.
+	 *
+	 * @return boolean
+	 */
+	public static function should_have_type( $events, $event ) {
+		if ( ! is_array( $events ) ) {
+			return false;
+		}
+
+		if ( empty( $event->timeslot ) ) {
+			return false;
+		}
+
+		$event_id = is_numeric( $event ) ? $event : $event->ID;
+		$index = false;
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 4 spaces but found 1 space

<sub>hash: 495a1db882bfbe1a22cf0ff6a0191435</sub>